src/components/Field_Input.tsx

```diff
@@ -0,0 +1,66 @@
+import Image from "next/image";
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  label?: string;
+  placeholder?: string;
+  icon?: string;
+  required?: boolean;
+}
+
+/*
+This is a component for the HTML `<input>' tag
+Input:
+  label: The label displayed on the text box
+  placeholder: The placeholder message displayed on the text box
+  icon: An optional Image path to display an .svg file
+Output:
+  A input text box that is compatible w/ React-hook-forms
+*/
+export default function Field_Input<T extends FieldValues = FieldValues>({
+  label,
+  placeholder = "",
+  icon = "",
+  ...props
+}: FormProps<T>) {
+  const { field } = useController(props);
+
+  placeholder == "" ? (placeholder = "Enter " + props.name) : "";
+
+  return (
+    <div className="relative mb-2 flex h-12 w-64 flex-row items-center justify-between rounded-lg border border-grey-300 px-3 shadow">
+      <div className=" absolute -top-2 left-2 flex flex-row items-center gap-0.5 rounded-full bg-[#FFF] px-1">
+        <label className="text-black text-xs font-semibold">
+          {label ?? props.name}
+        </label>
+        {props.rules?.required ? (
+          <span className="text-xs font-semibold text-[#FF0000]">*</span>
+        ) : (
+          ""
+        )}
+      </div>
```

Comment:
Since we use this a lot, you could refactor this out in a separate component. Check out https://blog.logrocket.com/using-react-children-prop-with-typescript/